media/src/main/java/io/wcm/handler/media/ui/ResourceMedia.java

```diff
@@ -83,6 +83,9 @@
   @RequestAttribute(injectionStrategy = InjectionStrategy.OPTIONAL)
   private String cssClass;
 
+  @RequestAttribute(injectionStrategy = InjectionStrategy.OPTIONAL)
+  private Boolean autoCrop;
```

Comment:
please add a documentation in the class javadoc comment above for this new parameter.

Reply:
https://github.com/wcm-io/wcm-io-handler/blob/develop/media/src/main/java/io/wcm/handler/media/ui/ResourceMedia.java#L57

Reply:
Oh I see, will do it, and add also the other missing ones